src/service/gateway.proto

```diff
@@ -13,8 +14,25 @@ enum close_state {
   close_state_dispute = 3;
 }
 
+enum error_code {
+  error_generic = 0;
```

Comment:
why prefix it all with `error_` if it's already in the error_code enum?

Reply:
no hard reason other than the error prefix makes it easier to convey the msgs are a control set on the client side